tests/Foundation/ApplicationTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use October\Rain\Foundation\Application;
+
+class ApplicationTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        $this->basePath = 'custom-path';
+        $this->app = new Application($this->basePath);
+    }
+
+    public function testPathMethods()
+    {
+        $this->assertEquals($this->app->pluginsPath(), $this->app->buildPath('/plugins'));
+        $this->assertEquals($this->app->themesPath(), $this->app->buildPath('/themes'));
+        $this->assertEquals($this->app->tempPath(), $this->app->buildPath('/storage/temp'));
+        $this->assertEquals($this->app->uploadsPath(), $this->app->buildPath('/storage/app/uploads'));
+        $this->assertEquals($this->app->mediaPath(), $this->app->buildPath('/storage/app/media'));
```

Comment:
I'd like to see testing for the buildPath method, since the other methods are basically small wrappers around it, it's more likely that it would break without breaking these tests.